### ISSUE TITLE
Intro section of Accessibility page is not accessible

### ISSUE THREAD

**Issue by onursenture** [primefaces/primereact#2764]
![unknown](https://user-images.githubusercontent.com/244957/163843798-cd4593a9-e1cc-42d4-b4a1-d6d8eeef2b4f.png)
